react/features/facial-recognition/facialExpressionsWorker.js

```diff
@@ -48,53 +36,40 @@ const window = {
 };
 
 onmessage = async function(message) {
-    switch (message.data.type) {
+    const { image, type } = message.data;
+
+    switch (type) {
     case INIT_WORKER : {
         modelsURL = message.data.url;
         if (message.data.windowScreenSize) {
             window.screen = message.data.windowScreenSize;
         }
+        if (self.useWASM) {
```

Comment:
Why not use the WASM backend unconditionally? The other code path will take care of converting the image if necessary, no?

Reply:
the webgl backend seems to be faster when it is available

Reply:
And I thought to use wasm only on browsers that cannot make use of WebGL in a worker